client/zeth/cli/zeth_mix.py

```diff
@@ -26,6 +26,7 @@
 @option("--eth-addr", help="Sender's eth address or address filename")
 @option("--eth-private-key", help="Sender's eth private key file")
 @option("--wait", is_flag=True, help="Wait for transaction to be mined")
+@option("--show-parameters", is_flag=True, help="Show parameters to the mixer")
```

Comment:
`Show parameters to the mixer` -> `Show parameters of the mix call`?

Reply:
reworded this

Reply:
You wrote:
> @option("--show-parameters", is_flag=True, help="Show the mixer parameters")

I haven't ran this code, but if I followed it correctly, this flag seems to be used to display the args of `zeth mix` - not the parameters of the `Mixer` contract. Did I miss something?

Reply:
It should dump the (json representation of) the `MixParameters` object, which is the in-memory version of the parameters passed to the mixer contract.

Reply:
I see, shall `Show the mixer parameters` be changed to `Show the mix call parameters` then?

Reply:
"mixer parameters" seems pretty unambiguous to me (i.e. "parameters to the mixer"), but we can reword it if you prefer.  "mix call parameters" alone could be slightly ambiguous, simply because we have client-side functions and commands called `mix`.

Reply:
This is a detail so I'm happy to keep "mixer params". For the record "mixers params" confuses me because: The Mixer is the contract name, hence "mixer parameters" makes me think about the parameters of its constructor and/or the value of the Mixer contract attributes (i.e. written in the Mixer contract's state). Wihtout looking at the code and just by the description of the flag I though I did a call to "get" the attributes values on the mixer to display them to the user here (i.e. Merkle tree depth, etc etc).